generators/app/templates/src/app/screens/Dashboard/screens/Home/index.js

```diff
@@ -12,11 +24,11 @@ function Home() {
           Edit <code>src/app/index.js</code> and save to reload.
         </p>
         <a className={styles.appLink} href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
+          Learn {foo}
         </a>
       </header>
     </div>
   );
 }
 
-export default Home;
+export default withProvider({ Context, reducer, initialState: INITIAL_STATE })(Home);
```

Comment:
I noticed we remove documentation from some components. Should we add a little bit of this component? Just explaining what it does and when to use it

Reply:
I think all the docs I removed are for components that aren't present anymore. I like the idea of adding documenttion for this component, but I rather do it in another card so we can close this. I added it to the backlog: https://trello.com/c/ficXXNHQ/241-agregar-documentacion-para-withprovider